TRANSCRIPTION.md

```diff
@@ -5,122 +5,88 @@
-| á | 166 | ✓ | ✓ |
-| í | 151 | ✓ | ✓ |
-| u | 144 | ✓ | ✓ |
-| h | 138 | ✓ | ✓ |
+| e | 179 | ✓ | ✓ |
+| á/a | 166 | ✓ | ✓ |
+| h | 153 | ✓ | ✓ |
+| í/i | 153 | ✓ | ✓ |
+| u | 147 | ✓ | ✓ |
 | m | 129 | ✓ | ✓ |
 | p | 119 | ✓ | ✓ |
+| ʔ | 119 | ✓ | ✓ |
+| w | 104 | ✓ | ✓ |
 | r | 97 | ✓ | ✓ |
-| w | 96 | ✓ | ✓ |
-| ʔ | 81 | ✓ | ✓ |
 | l | 74 | ✓ | ✓ |
-| ú | 68 | ✓ | ✓ |
+| ú/u | 68 | ✓ | ✓ |
 | t | 66 | ✓ | ✓ |
-| j | 59 | ✓ | ✓ |
-| o | 57 | ✓ | ✓ |
-| é | 53 | ✓ | ✓ |
-| ó | 46 | ✓ | ✓ |
+| j | 60 | ✓ | ✓ |
+| o | 59 | ✓ | ✓ |
+| é/e | 54 | ✓ | ✓ |
+| ó/o | 46 | ✓ | ✓ |
 | d | 45 | ✓ | ✓ |
+| ʃ | 43 | ✓ | ✓ |
 | b | 39 | ✓ | ✓ |
-| ʃ | 36 | ✓ | ✓ |
-| ɾ | 32 | ✓ | ✓ |
+| ɾ | 33 | ✓ | ✓ |
 | aa | 29 | ✓ | ✓ |
 | tʃ | 27 | ✓ | ✓ |
 | ii | 20 | ✓ | ✓ |
+| pʰ | 20 | ✓ | ✓ |
 | s | 20 | ✓ | ✓ |
-| à | 18 | ✓ | ✓ |
+| à/a | 18 | ✓ | ✓ |
 | g | 17 | ✓ | ✓ |
 | v | 17 | ✓ | ✓ |
-| ? | 16 | ? | ? |
 | ee | 16 | ✓ | ✓ |
-| ( | 13 | ? | ? |
 | ts | 13 | ✓ | ✓ |
-| ) | 12 | ? | ? |
+| ɲ | 13 | ✓ | ✓ |
 | a: | 12 | ✓ | ✓ |
 | i: | 12 | ✓ | ✓ |
-| ph | 12 | ✓ | ✓ |
-| nh | 10 | ✓ | ✓ |
-| ì | 10 | ✓ | ✓ |
-| ù | 9 | ✓ | ✓ |
-| 'a | 8 | ✓ | ✓ |
+| tsʰ | 10 | ✓ | ✓ |
+| ì/i | 10 | ✓ | ✓ |
+| kʰ | 9 | ✓ | ✓ |
+| ù/u | 9 | ✓ | ✓ |
 | dz | 8 | ✓ | ✓ |
 | e: | 8 | ✓ | ✓ |
-| pʰ | 8 | ✓ | ✓ |
 | y | 8 | ✓ | ✓ |
-| è | 8 | ✓ | ✓ |
-| é: | 8 | ✓ | ✓ |
-| kh | 7 | ✓ | ✓ |
-| tsʰ | 7 | ✓ | ✓ |
+| è/e | 8 | ✓ | ✓ |
+| é:/eː | 8 | ✓ | ✓ |
 | uu | 7 | ✓ | ✓ |
 | oo | 6 | ✓ | ✓ |
 | tt | 6 | ✓ | ✓ |
 | _ | 5 | ✓ | ✓ |
 | dʒ | 5 | ✓ | ✓ |
-| rh | 5 | ? | ? |
+| rʰ | 5 | ✓ | ✓ |
+| tʃʰ | 5 | ✓ | ✓ |
 | t̪ | 5 | ✓ | ✓ |
-| wh | 5 | ? | ? |
-| mh | 4 | ✓ | ✓ |
-| tʃh | 4 | ? | ? |
-| í: | 4 | ✓ | ✓ |
-| 'i | 3 | ✓ | ✓ |
-| 'u | 3 | ✓ | ✓ |
-| [ | 3 | ? | ? |
-| ] | 3 | ? | ? |
-| ch | 3 | ✓ | ✓ |
+| mʰ | 4 | ✓ | ✓ |
+| í:/iː | 4 | ✓ | ✓ |
 | f | 3 | ✓ | ✓ |
-| hw | 3 | ? | ? |
-| tsh | 3 | ? | ? |
-| á: | 3 | ✓ | ✓ |
-| íí | 3 | ✓ | ✓ |
+| á:/aː | 3 | ✓ | ✓ |
+| íí/iː | 3 | ✓ | ✓ |
 | ñ | 3 | ✓ | ✓ |
 | ĩ | 3 | ✓ | ✓ |
-| š | 3 | ? | ? |
-| 'o | 2 | ✓ | ✓ |
+| ʒ | 3 | ✓ | ✓ |
 | c | 2 | ✓ | ✓ |
-| kʰ | 2 | ✓ | ✓ |
-| lh | 2 | ? | ? |
-| th | 2 | ✓ | ✓ |
+| tʰ | 2 | ✓ | ✓ |
+| ú:/uː | 2 | ✓ | ✓ |
 | ũ | 2 | ✓ | ✓ |
-| ɲ | 2 | ✓ | ✓ |
-| ' | 1 | ? | ? |
-| ') | 1 | ? | ? |
-| 'e | 1 | ✓ | ✓ |
-| 'í | 1 | ✓ | ✓ |
-| (h | 1 | ? | ? |
-| )h | 1 | ? | ? |
-| Ih | 1 | ? | ? |
-| J | 1 | ? | ? |
-| hn | 1 | ? | ? |
-| hɲ | 1 | ? | ? |
-| hɾ | 1 | ? | ? |
-| jβ | 1 | ? | ? |
-| ll | 1 | ? | ? |
+| ʎ | 2 | ✓ | ✓ |
+| β | 2 | ✓ | ✓ |
+| lː | 1 | ✓ | ✓ |
 | n̥ | 1 | ✓ | ✓ |
 | u: | 1 | ✓ | ✓ |
-| éh | 1 | ? | ? |
-| éː | 1 | ✓ | ✓ |
-| ê | 1 | ✓ | ✓ |
-| í́ | 1 | ✓ | ✓ |
-| ñh | 1 | ✓ | ✓ |
-| ô | 1 | ✓ | ✓ |
-| ú: | 1 | ✓ | ✓ |
-| úú | 1 | ✓ | ✓ |
-| čh | 1 | ? | ? |
-| ǰ | 1 | ? | ? |
-| ɻh | 1 | ? | ? |
-| ʔh | 1 | ✓ | ✓ |
-| ʔʒ | 1 | ✓ | ✓ |
-| β | 1 | ✓ | ✓ |
+| éː/eː | 1 | ✓ | ✓ |
+| ê/e | 1 | ✓ | ✓ |
+| ô/o | 1 | ✓ | ✓ |
+| ɪ | 1 | ✓ | ✓ |
+| ɻ | 1 | ✓ | ✓ |
+| ʰn | 1 | ✓ | ✓ |
```

Comment:
pre-aspirated n, where does this come from? from `nh`? If so, did you confirm it is pre-aspirated and not rather simply devoiced, as in most cases of usage?

Reply:
For both this and `ʰɲ`, the raw source makes it clear that it is indeed a glottal fricative in the coda of the previous syllable (it is hard to say much, because the second lexeme has no obvious cognates, and this one has a single one with a tone filling the place of the fricative).

It is still a bit strange because this coda doesn't seem to follow the phonotactics suggested by the other lexemes, but as it is in the source (even with a `j` grapheme, from Spanish) I'll replace them.